dataScripts/tessTrain/tessTrain.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+# Tested on a bare install of Ubuntu 22.04.3 LTS - 02/27/2024
+# Author: Charles@CharlesBock.com - github.com/Penguin2600
+
+# Color Stuff
+BGreen='\033[1;32m'
+NC='\033[0m'
+
+greentext () {
+   echo -e "\n${BGreen}### $1 ${NC}\n"
+}
+
+# Build tesseract
+# Upstream Docs: https://tesseract-ocr.github.io/tessdoc/Compiling-%E2%80%93-GitInstallation#installing-with-autoconf-tools
+greentext "Installing Deps and Creating File Structure"
+
+# Dont polute the directory
+mkdir -p ./tess
+cd tess
+pwd
+
+# Get Deps
+sudo apt-get install libicu-dev libpango1.0-dev libcairo2-dev
+sudo apt-get install automake ca-certificates g++ git libtool libleptonica-dev make pkg-config
+sudo apt-get install libpango1.0-dev libleptonica-dev
```

Comment:
I think it would make sense to extract this into the README as a `## training tesseract` section. 

I would split this script into two parts - a `setup.sh` script (also mention it in the README in the setup instructions) and a `train.sh` script that takes in a ground truth path.

Reply:
Totally, I had the same thought. one will likely run once while the other may need many runs.